src/appkit.cs

```diff
@@ -11376,8 +11652,12 @@ partial interface NSPasteboard // NSPasteboard does _not_ implement NSPasteboard
 		NSString NSPasteboardTypeMultipleTextSelection { get; }
 
 		[Field ("NSPasteboardTypeFindPanelSearchOptions")]
+		[Availability (Deprecated = Platform.Mac_10_14, Message = "Use 'NSPasteboardTypeTextFinderOptions' instead.")]
 		NSString NSPasteboardTypeFindPanelSearchOptions { get; }
 
+		[Mac (10, 7), Field ("NSPasteboardTypeTextFinderOptions")]
+		NSString PasteboardTypeTextFinderOptions { get; }
```

Comment:
consistency, other constants where (sadly) exposed with the `NS` prefix.
also the deprecation message, just above, use the `NS` prefix